### PR TITLE
Refactoring and avoid try Reconnection when 2 exception at connect and Tempory fix for multiple boss and worker group supports

### PR DESCRIPTION
**Pull request by Yasara123**

### CODE REVIEW

File: component/src/main/java/org/wso2/extension/siddhi/io/http/source/HttpSource.java [hunk 1]
```diff
@@ -214,7 +214,8 @@ public void disconnect() {
 
     @Override
     public void destroy() {
-        this.httpConnectorRegistry.stopHttpServerConnectorController();
+        // TODO: 7/26/17 Until fix for multiple worker and boss thread loop group
+        //this.httpConnectorRegistry.stopHttpServerConnectorController();
```
Comment (dilini-muthumala) [inline component/src/main/java/org/wso2/extension/siddhi/io/http/source/HttpSource.java]: @Yasara123 When can we expect this fix? If it's not going to happen any time soon, better to have a GIT issue created for this, so we will not miss to fix this.

File: component/src/main/java/org/wso2/extension/siddhi/io/http/util/HttpConstants.java [hunk 2]
```diff
@@ -106,5 +106,6 @@ private HttpConstants() {
     public static final String EMPTY_ISAUTH = "false";
     public static final String ATHORIZATION_HEADER = "Authorization";
     public static final String CARBON_SECURITY_CONFIGURATION = "CarbonSecurityConfig";
-    public static final String MESSAGEPROCESSOR_ID = "Siddhi HTTP-message-processor";
+    public static final String MESSAGEPROCESSOR_ID = "messageProcessorId";
```
Comment (dilini-muthumala) [inline component/src/main/java/org/wso2/extension/siddhi/io/http/util/HttpConstants.java]: MESSAGEPROCESSOR_ID better be MESSAGE_PROCESSOR_ID....WDYT?

File: component/src/main/java/org/wso2/extension/siddhi/io/http/source/HttpSource.java (hunk 1 above)
Comment (Yasara123) [inline component/src/main/java/org/wso2/extension/siddhi/io/http/source/HttpSource.java]: Fix in https://github.com/wso2-extensions/siddhi-io-http/issues/17

File: component/src/main/java/org/wso2/extension/siddhi/io/http/util/HttpConstants.java (hunk 2 above)
Comment (Yasara123) [inline component/src/main/java/org/wso2/extension/siddhi/io/http/util/HttpConstants.java]: Fix in 8757397

**Pull request merged**